exercise4/flattenArray.js

```diff
@@ -0,0 +1,3 @@
+function flattenArray(array) {
+  return array.flat(Number.POSITIVE_INFINITY);
```

Comment:
Using Number.POSITIVE_INFINITY as the depth parameter ensures that all nested arrays are flattened, which is a clever way to handle arrays of arbitrary depth.

Reply:
your flattenArray function seems well-implemented. Good job